benchmarks/urls.py

```diff
@@ -1,22 +1,40 @@
 from django.urls import path
+import functools
 
 from .views import index, user, model, competition
 
 urlpatterns = [
     # index
-    path('', index, name='index'),
```

Comment:
let's use a for loop over `['vision', 'language']` instead of a gigantic hand-written list